src/Pim/Component/Catalog/Normalizer/Indexing/Product/PropertiesNormalizer.php

```diff
@@ -77,4 +97,29 @@ public function supportsNormalization($data, $format = null)
     {
         return $data instanceof ProductInterface && 'indexing' === $format;
     }
+
+    /**
+     * @param ProductInterface $product
+     *
+     * @return array
+     */
+    protected function getAssociatedProducts(ProductInterface $product)
+    {
+        $isAssociated = [];
+
+        $associations = $this->associationRepository->getAssociationsContainingProduct($product);
+
+        foreach ($associations as $association) {
+            $associationTypeCode = $association->getAssociationType()->getCode();
+            $ownerIdentifier = $association->getOwner()->getIdentifier();
+
+            if (!isset($isAssociated[$associationTypeCode])) {
+                $isAssociated[$associationTypeCode] = [];
+            }
```

Comment:
Yes I do, I have an error if I don't do that, PHP does not accept that I directly create a double index. I though it could work but it seems it doesn't.